client_tls.go

```diff
@@ -45,11 +45,14 @@ func appendCert(path string, pool *x509.CertPool) {
 	log.Debugf("Appended cert from '%s'", path)
 }
 
-// NewCertPool loads PEM certificates from given path and returns them in a way that is usable at TLS() as RootCAs.
+// NewCertPool initiates system's cert pool then loads PEM certificates from given path and returns them in a way that is usable at TLS() as RootCAs.
```

Comment:
How about "NewCertPool **loads** system's cert pool **and adds** PEM certificates"?